data/generateData.js

```diff
@@ -60,14 +61,17 @@ async function parseSheet(sheet) {
   };
 }
 
-function getStatsByPlayer(acc, { players }, index, rounds) {
+function getStatsByPlayer(maxRoundsPerPlayer, acc, { players }, index, rounds) {
   players.forEach(({ name, scores, total }) => {
     const player = acc[name];
 
     const cumScore = idx(player, 'cumScore', 0) + total;
     const roundsPlayed = idx(player, 'roundsPlayed', 0) + 1;
     const avgScore = getAvg(cumScore, roundsPlayed);
 
+    // only include first X rounds for a given player in their stats
+    if (roundsPlayed > maxRoundsPerPlayer) return;
```

Comment:
This isn't 'nam!

```suggestion
    if (roundsPlayed > maxRoundsPerPlayer) {
      return;
    }
```

Reply:
There are lint rules in this repo and this style passed those rules. Wont fix

Reply:
sadpanda

Guess I've gotta add a lint rule to prevent abominations like this, then!

Reply:
https://eslint.org/docs/rules/curly